File: packages/jsii-rosetta/test/translations/statements/vararg_any_call.cs
public void Test(params object[] _args)
{
}

Test(new Dictionary<string, object> { { "Key", "Value" }, { "also", 1337 } });

Test(new Dictionary<string, string> { { "Key", "Value" } }, new Dictionary<string, int> { { "also", 1337 } });

File: packages/jsii-rosetta/test/translations/statements/vararg_any_call.go
func test(..._args interface{}) {
}

test(map[string]interface{}{
	"Key": jsii.String("Value"),
	"also": jsii.Number(1337),
})

test(map[string]*string{
	"Key": jsii.String("Value"),
}, map[string]*f64{
	"also": jsii.Number(1337),
})
